### ISSUE TITLE
Time boolean

### ISSUE THREAD

**Issue by UndermanReal** [TurboWarp/extensions#489]
![Button](https://github.com/TurboWarp/extensions/assets/131917462/99763adc-8208-4712-b52b-c1030098687c)
Something like this could be very useful (pretend the one in the image is a Boolean and not a block)

**Comment by Skyhigh173** [comment]
is this connected to (timer) block and just (timer) > (5)?

**Comment by LukeManiaStudioss** [comment]
That as a Boolean is ambiguous. Does it tick true for 1 frame every 5 seconds?

**Comment by SurvExe1Pc** [comment]
that seems useful, would it work for multiple at a time?

**Comment by Skyhigh173** [comment]
> No, let me rephrase it. You input something in the boolean, and it will have a timer for your input, which is the amount of seconds it waits until it's true. This would be used in wait until blocks, where if you want to wait for a certain amount of seconds while also checking if another thing is true


So `wait until <> or wait () seconds` ?

**Comment by True-Fantom** [comment]
@Skyhigh173 Most likely... There is a similar implementation in penguinmod and it is quite convenient:
![Скриншот 02-06-2023 110932](https://github.com/TurboWarp/extensions/assets/116278271/af518a3b-a5b4-4be5-a454-26b36c7f5617)

**Comment by SharkPool-SP** [comment]
tada

<img width="579" height="291" alt="Image" src="https://github.com/user-attachments/assets/ebf790a9-c891-4c32-a22d-3cb66ac56894" />
